interface_specifications/pe_openapi/apis/v1/b0_pe_post.md

```diff
@@ -0,0 +1,23 @@
+### Thread POST
+
+`POST /pe/v1/`
```

Comment:
Given we have removed the thid from the URLs and tucked them into the request bodies, it is probably cleaner to have this endpoint moved to /'threads, so that it is outside of the path hierarchy and does not have overlap anymore more with endpoints like /pe/v1/definitions in the hierarchy

Reply:
fixed